docs/faq.md

```diff
@@ -224,6 +224,30 @@ Commandes utiles:
 
 Source du plugin: <https://github.com/NotSqrt/mattermost-integration-gitlab>
 
+### Mattermost Gitlab-CI integration
+
+Se rendre sur <https://mattermost.fabrique.social.gouv.fr/default/integrations/incoming_webhooks/add> et créer un webhook en sélectionant le canal sur lequel on souhaite publier les notifications. Récupérer l'url du webhook qui contiendra le token. Se rendre sur la page de configuration CI/CD du projet sur GitLab, example: https://gitlab.factory.social.gouv.fr/SocialGouv/emjpm/-/settings/ci_cd (remplacer emjpm par le nom du projet), et ajouter une variable intitulée `MATTERMOST_WEBHOOK` dans la sections `variables` et qui contiendra l'url du webhook.
+
+Dans le fichier `.gitlab-ci` à la racine du projet
+```yaml
+Ma notification mattermost:
+  image: registry.gitlab.factory.social.gouv.fr/devthejo/bash-curl:1.2.0
```

Comment:
We should use a common curl image

```suggestion
  image: registry.gitlab.factory.social.gouv.fr/socialgouv/docker/curl:2.9.0
```

or 

```suggestion
  image: curlimages/curl:7.74.0
```

Reply:
j'ai créé `registry.gitlab.factory.social.gouv.fr/devthejo/bash-curl:1.2.0` car j'avais besoin du bash pour mon script qui extrait la release du changelog et il n'y a pas de bash dans `curlimages/curl:7.74.0`